packages/featuregates/bundle/config/upstream/tanzu-featuregates-manager.yaml

```diff
@@ -61,6 +61,8 @@ spec:
       tolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/control-plane
```

Comment:
I'm asking cause I have a similar task for another package. Should we maintain ```node-role.kubernetes.io/master``` for now or we should already substitute this line with ```node-role.kubernetes.io/control-plane```?

Reply:
@maralavi, I'm also not sure. I had a similar question: why not just remove the old `node-role.kubernetes.io/master` now so we don't have to remove it later?

The reason I kept both is because I don't know if the Tanzu version that will contain this PR's change has a possibility of running on a Kubernetes version older than 1.23. If running with an older Kubernetes version is a possibility, then my change will break backward compatibility. That seems bad. So, since we can have multiple tolerations, I thought it best to keep both the "master" and "control-plane" tolerations.

Reply:
> The reason I kept both is because I don't know if the Tanzu version that will contain this PR's change has a possibility of running on a Kubernetes version older than 1.23. If running with an older Kubernetes version is a possibility, then I don't want my change to break backward compatibility.

It should be safe to remove old toleration as they are both valid from k8s v1.20 to allow this transition.

Reply:
> It should be safe to remove old toleration as they are both valid from k8s v1.20 to allow this transition.

Everyone else submitting and raising PRs are doing as we were originally instructed adding a new toleration (not replacing). So I'll keep both.